ui/src/app/applications/components/application-pod-view/pod-view.scss

```diff
@@ -250,3 +250,7 @@ $pod-age-icon-clr: #ffce25;
 .tippy-content {
     text-align: left;
 }
+
+.tippy-popper {
+    z-index: 1 !important;
+}
```

Comment:
I'm not up on SCSS... is this style scoped at all? Should we be worried about this style affecting tippy tooltips in other contexts?

Reply:
Used the tippy z-index prop to handle this.
doc for tippy.js : https://atomiks.github.io/tippyjs/v5/all-props/